Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/FixedWidthFlatFile.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System.ComponentModel;
+using System.Globalization;
+using Frends.FixedWidthFlatFile.Parse.Definitions;
+
+namespace Frends.FixedWidthFlatFile.Parse;
+
+public static class FixedWidthFlatFile
+{
+    /// <summary>
+    /// Parse Fixed Width data to object. [Documentation](https://tasks.frends.com/#frends-tasks/Frends.FixedWidthFlatFile.Parse)
+    /// </summary>
+    /// <param name="input"></param>
+    /// <param name="options"></param>
+    /// <returns>Object { List&lt;Dictionary&lt;string Key, dynamic Value&gt;&gt; Data }</returns>
+    public static Result Parse([PropertyTab] Input input, [PropertyTab] Options options)
+    {
+        var inputRows = new List<string>();
+        var headers = new List<string>();
+        var outputData = new List<Dictionary<string, dynamic>>();
+
+        //== Read input data ==/
+        using (var reader = new StringReader(input.FlatFileContent))
+        {
+            inputRows = ReadLinesToList(reader);
+        }
+
+        //== Parse header row ==/
+        switch (input.HeaderRow)
+        {
+            case HeaderRowType.Delimited:
+                char headerDelimiter = Convert.ToChar(input.HeaderDelimiter);
+                headers = SplitToList(inputRows.First(), headerDelimiter);
+                inputRows.RemoveAt(0);
+                break;
+            case HeaderRowType.FixedWidth:
+                headers = SplitToList(inputRows.First(), input.ColumnSpecifications);
+                inputRows.RemoveAt(0);
+                break;
+        }
+        if (headers.Count > 0)
+        {
+            // add header values as name if not set in column specification
+            var index = 0;
+            foreach (var header in headers)
+            {
+                if (string.IsNullOrEmpty(input.ColumnSpecifications[index].Name))
+                    input.ColumnSpecifications[index].Name = header;
+                index++;
+            }
+        }
+
+        //== Parse data rows ==/
+
+        // Skip rows?
+        if (options.SkipRows)
+        {
+            if (options.SkipRowsFromTop > 0)
+            {
+                // skipping more rows that exist?
+                if (options.SkipRowsFromTop >= inputRows.Count)
+                    inputRows.Clear();
+                else
+                    inputRows = inputRows.Skip(options.SkipRowsFromTop).ToList();
+            }
+            if (options.SkipRowsFromBottom > 0)
+            {
+                // skipping more rows that exist?
+                if (options.SkipRowsFromBottom >= inputRows.Count)
+                    inputRows.Clear();
+                else
+                    inputRows = inputRows.Take((inputRows.Count - options.SkipRowsFromBottom)).ToList();
+            }
+        }
+        //== Process data rows ==/
+        foreach (var dataRow in inputRows)
+        {
+            outputData.Add(ParseDataRow(dataRow, input.ColumnSpecifications));
```

Comment:
So here it would be nice if the input.ColumnSpecifications could be typed for example to string.